core/chains/evm/txmgr/orm_test.go

```diff
@@ -105,7 +107,7 @@ func TestORM_EthTransactions(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 3, count)
 
-	txs, count, err := orm.EthTransactions(0, 100)
+	txs, count, err := orm.EthTransactions(0, 100, pg.WithParentCtx(context.Background()))
```

Comment:
Is it actually necessary to pass an explicit param with a background context? Why is the param optional in that case?

Reply:
For this test case it is not necessary. But in general, the optional opts allows the developer the ability to make the ORM call with a different queryer, a different context, a different timeout, or define their own function which returns pg.Q